app.js

```diff
@@ -10,21 +10,26 @@ function findByOdsCode(data, odsCode) {
   console.time(timerMsg);
 
   const bookingSystemData = fs.readFileSync('./input/booking.json');
-  const scriptData = fs.readFileSync('./input/scripts.json');
+  const recordsSystemData = fs.readFileSync('./input/records.json');
+  const scriptSystemData = fs.readFileSync('./input/scripts.json');
   const gpData = fs.readFileSync('./input/gp-data.json');
 
   const gps = JSON.parse(gpData);
   const bookingSystems = JSON.parse(bookingSystemData);
-  const scriptSystems = JSON.parse(scriptData);
+  const scriptSystems = JSON.parse(scriptSystemData);
+  const recordsSystems = JSON.parse(recordsSystemData);
 
   const merged = gps.map((gp) => {
     // eslint-disable-next-line no-param-reassign
```

Comment:
the map function is getting a bit long and doing three things, would prefer to see the blocks of logic in the body split into smaller functions that do one thing